src/i18n/locales/en/translation.json

```diff
@@ -818,6 +818,27 @@
         "label": "Amount (Satoshi)"
       }
     },
+    "send_to_bitcoin_address": {
+      "title": "Send to bitcoin address",
+      "recipient": {
+        "label": "Recipient"
+      },
+      "amount": {
+        "label": "Amount (sats)"
+      },
+      "network_fee": {
+        "label": "Network fee"
+      },
+      "service_fee": {
+        "label": "Service fee"
```

Comment:
Should we explicitly say "Alby service fee" here? I think it's easier to differentiate for users

Reply:
I actually think splitting this is too complicated 
the user has no choice in setting either fee, so why not just have one amount the user has to pay?

Reply:
I'd leave it in for now as (especially during high fee environments) users know what they pay for.